app/jobs/create_developer_badge_job.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDeveloperBadgeJob < ApplicationJob
```

Comment:
Let's remove this job, test, and backfill in this PR since it will be merged in after #776.